.gitignore

```diff
@@ -109,3 +109,8 @@ Makefile
 ### Linux ###
 *~
 *.autosave
+# Adding in-project build folders to git-ignore.
+# Any folder starting with the name build* is ignored
+# This is useful for cross-platform developers who might
+# want folders such as build-w or build-l for respective platforms
+build*/
```

Comment:
I use cogutils/build-w and cogutils/build-l for my build folders where cogutils is the project root.  It works well with my text/code editor, which only shows me files in the project root.  I don't foresee any non-build folders being created with the "build" string prefixed.